### PR TITLE
Updated Finnish translation

### PR DESCRIPTION
**Pull request by river-is-mmm**
- added new entries
- translated link hover tooltip

### CODE REVIEW

File: src/locale/fi.json [hunk 1]
```diff
@@ -193,5 +193,7 @@
 		"searchBar": "tämä ei ole oikea hakupalkki",
 		"snedHelp": "lähetä apuaaa",
 		"perfection": "täydellisyys"
```
Comment (Pizzacus) [inline src/locale/fi.json]: You're gonna need a `,` here

**Comment by Pizzacus** [comment]
Also, geez, you're fast

File: src/locale/fi.json (hunk 1 above)
Comment (river-is-mmm) [reply src/locale/fi.json]: Damn, let me add that real quick

**Comment by river-is-mmm** [comment]
There, added

**Comment by Pizzacus** [comment]
Looks good, thanks!

**Pull request merged**